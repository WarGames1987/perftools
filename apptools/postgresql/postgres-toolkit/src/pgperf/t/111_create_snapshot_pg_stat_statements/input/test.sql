SELECT pg_stat_statements_reset();

SELECT pgperf.create_snapshot_pg_stat_statements(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;

SELECT pgperf.create_snapshot_pg_stat_statements(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;

SELECT pgperf.delete_snapshot_pg_stat_statements(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;

SELECT pgperf.delete_snapshot_pg_stat_statements(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;
