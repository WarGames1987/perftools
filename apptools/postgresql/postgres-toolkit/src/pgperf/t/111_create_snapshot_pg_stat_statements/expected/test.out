SELECT pg_stat_statements_reset();
 pg_stat_statements_reset 
--------------------------
 
(1 row)

SELECT pgperf.create_snapshot_pg_stat_statements(0);
 create_snapshot_pg_stat_statements 
------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;
 count 
-------
     1
(1 row)

SELECT pgperf.create_snapshot_pg_stat_statements(1);
 create_snapshot_pg_stat_statements 
------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;
 count 
-------
     4
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_statements(0);
 delete_snapshot_pg_stat_statements 
------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;
 count 
-------
     3
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_statements(1);
 delete_snapshot_pg_stat_statements 
------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_statements;
 count 
-------
     0
(1 row)

